### ISSUE TITLE
Clarification on Performance Benefits of onKoinStartup vs. startKoin

### ISSUE THREAD

**Issue by AhmedAbidii** [InsertKoinIO/koin#2002]
Hello,

In the [AndroidX Startup: Optimized Loading blog post](https://blog.insert-koin.io/koin-4-0-official-release-f4827bbcfce3), it's mentioned that using onKoinStartup:

"reduces load times by up to 40%, improving performance for applications using Koin."

However, in the [Koin Android documentation](https://insert-koin.io/docs/reference/koin-android/start), it states that switching from onKoinStartup to the regular startKoin:

"can result in over 30% of startup time gained."

This seems contradictory. Since onKoinStartup was introduced in Koin 4.0 to optimize startup performance, shouldn't it provide better performance than the regular startKoin? Could you please clarify the expected behavior?

Thank you!

**Comment by blazmag** [comment]
The direct quote is:
> "Gain over from onKoinStartup to regular startKoin can go over 30% of time gained, for startup time." 

My 5 cents, some parts of the documentation would greatly benefit from a rewrite by a native english speaker. I think its just badly phrased and I read this as "if you use onKoinStartup you gain 30% lower start times vs startKoin" but I hope one of the devs can confirm.

**Comment by arnaudgiuliani** [comment]
You are all welcome to help with documentation :) Gain part is rewritten, and should be more part of some benchmark article.

**Comment by Nek-12** [comment]
Besides, that statement is not correct at all. Using onKoinStartup does not provide 30% speed improvement, it just shifts the measurement over to the place that isn't really captured by the author's benchmarks

**Comment by arnaudgiuliani** [comment]
Yep sure, other comparison will come with startKoin & androidx solution
